build/Build.cs

```diff
@@ -82,7 +82,7 @@ class Build : NukeBuild
                 .SetProjectFile(Solution)
                 .SetConfiguration(Configuration)
                 .SetLoggers("trx")
-                .SetVerbosity(DotNetVerbosity.Normal)
+                .SetVerbosity(DotNetVerbosity.normal)
```

Comment:
This affects me on a primal level, but an API's an API :P

Reply:
Indeed. Nuke (the library which defines this) is not my favourite piece of open source software